internal/models/results.go

```diff
@@ -18,6 +18,13 @@ type PickleResult struct {
 	StartedAt time.Time
 }
 
+// PickleAttachment ...
+type PickleAttachment struct {
```

Comment:
Is this type needed, or can we use `godog.Attachment` instead?

Reply:
See my previous comments messages.Attachment should remain and internal class.